README.md

```diff
@@ -177,6 +177,6 @@ cmake --build build/Debug --parallel
 
 I will write more extensively about the serialization format used by DataTamer, but for the time being I
 created a single header file without external dependencies that you can just copy into your project:
-[data_tamer_parser.hpp](data_tamer/include/data_tamer_parser)
+[data_tamer_parser.hpp](data_tamer_cpp/include/data_tamer_parser)
```

Comment:
```suggestion
[data_tamer_parser.hpp](data_tamer_cpp/include/data_tamer_parser/data_tamer_parser.hpp)
```

I think this should just link directly to the header file. Other than this, LGTM!

Reply:
Sure. Thanks for the correction